engines/e_afalg.c

```diff
@@ -24,11 +24,17 @@
 #include "internal/nelem.h"
 
 #include <sys/socket.h>
-#include <linux/version.h>
+
+#ifdef __wasi__
```

Comment:
Unless this is critical, I'd not bother.  This engine is deprecated.

Reply:
ok, I disabled build of the engine in my repo and tests still are passing (https://github.com/loganek/webassembly-curl-ssl/commit/4804c020a7bf47815633b529b47077f0b06f6ae5)